reference/forms/types/options/compound_type.rst.inc

```diff
@@ -0,0 +1,8 @@
+compound
+~~~~~~~~
+
+**type**: ``boolean`` **default**: ``false``
+
+This option specifies if the type is compound. This is independent of whether
```

Comment:
You are right! I'm waiting for the general fix of this issue: https://github.com/symfony/symfony-docs/issues/5213